packages/flutter/lib/src/rendering/editable.dart

```diff
@@ -196,12 +197,24 @@ class RenderEditable extends RenderBox {
     markNeedsSemanticsUpdate();
   }
 
+  /// This is used for updating the text in the text field in the event
```

Comment:
add a single line description in a separate paragraph above this one. (The dartdocs use the first paragraph as the brief description summary in indexes.)